docs/dashboard/configuration/shipping.md

```diff
@@ -53,6 +53,21 @@ These two methods cover most of the basic use cases, for example:
 * Increasing the shipping price for heavy orders.
 
 
+### ZIP Codes
```

Comment:
We should rename this to "postal codes" both in docs and in the UI. "ZIP" is a specific postal code system used by the US and the name comes from their "Zone Improvement Plan" program.

Reply:
Merged in https://github.com/mirumee/saleor-dashboard/pull/932